src/components/crownConfig.vue

```diff
@@ -1,5 +1,6 @@
 <template>
   <div class="crown-config" :style="style" v-if="showCrown">
+    <slot/>
```

Comment:
Decided to introduce a slot to add component specific buttons (in this case, the add lane buttons).